src/main/groovy/com/mini/asaas/utils/validators/ValidatorCpfCnpj.groovy

```diff
@@ -0,0 +1,67 @@
+package com.mini.asaas.utils.validators
+
+class ValidatorCpfCnpj {
+
+    static boolean isValidCpfCnpj(String cpfCnpj) {
+        return isValidCpf(cpfCnpj) || isValidCnpj(cpfCnpj)
+    }
+
+    private static boolean isValidCpf(String cpf) {
+        if (!cpf.matches("\\d{11}") || cpf.matches("(\\d)\\1{10}")) {
+            return false
+        }
+
+        int[] digits = cpf.collect { it as int - '0' as int }
```

Comment:
pra que serve o `- '0' as int` ?

`it` é um número, aí fazemos - 0, não vai continuar igual?

Reply:
A intenção era garantir que o caractere ia ser tratado como um numérico, mas pesquisei sobre Automatic Data Type Conversion do Groovy e vi que isso pode ser removido pq em operações a conversão automática é suportada